services/network.proto

```diff
@@ -0,0 +1,13 @@
+message ServerIdentity{
+    required bytes public = 1;
+    required bytes id = 2;
```

Comment:
Should ID be required ? Because it can be deterministically generated out of the public and address, I'm not sure. The protobuf library puts it as required if it's not a pointer or it does not have the `protobuf:"opt"` struct tag field.